docs/xtrabackup-versioning-scheme.md

```diff
@@ -0,0 +1,9 @@
+### Percona XtraDB Cluster 5.7 versioning scheme:
+
+Here is what the numbers in the product version specification mean:
+
+For example, in Percona XtraDB Cluster 5.7.39-31.61
+
+* `5.7.39-31` - the version of Percona Server for MySQL that Percona XtraDB Cluster is based on
```

Comment:
This is true for 8.0. For 5.7, '31' is the WSREP-API version.
Please check https://confluence.percona.com/pages/viewpage.action?pageId=104088973